pkg/server/server.go

```diff
@@ -162,6 +161,36 @@ func NewReplicationServer(
 	}
 
 	if options.Sync.Enable {
+		signer, err := blockchain.NewPrivateKeySigner(
```

Comment:
Will need to figure out a tidier way to get the Domain Separator into sync worker